srcpkgs/ol/template

```diff
@@ -1,22 +1,24 @@
 # Template file for 'ol'
 pkgname=ol
-version=2.3.6
+version=2.4
 revision=1
 build_style=gnu-makefile
+make_use_env=yes
 hostmakedepends="xxd"
 short_desc="Purely functional dialect of Lisp"
 maintainer="rc-05 <rc23@email.it>"
 license="LGPL-3.0-or-later, MIT"
 homepage="https://yuriy-chumak.github.io/ol/"
 changelog="https://raw.githubusercontent.com/yuriy-chumak/ol/master/doc/CHANGELOG.md"
 distfiles="https://github.com/yuriy-chumak/ol/archive/${version}.tar.gz"
-checksum=5a7b8ff5eeba2db63ba8d409c6ebc3af5f9cb21d14a2ec1a279d8137298360ef
+checksum=019978ddcf0befc8b8de9f50899c9dd0f47a3e18cf9556bc72a75ae2d1d965d4
 
-case "$XBPS_TARGET_MACHINE" in
-	i686*|x86_64*) makedepends+=" gcc-multilib";;
-esac
-
-CFLAGS="${CFLAGS/-D_FORTIFY_SOURCE=2/} -lm -DOLVM_FFI -DHAS_SOCKETS -DHAS_DLOPEN"
+make_build_args="HAS_SOCKETS=1 HAS_DLOPEN=1 OLVM_FFI=1"
```

Comment:
put this above with make_use_env

Reply:
why? that's an important line with build options to not be lost inside urls and descriptions.
imho.

Reply:
> buildtesting with proot/qemu isn't very useful. it's better to do it like it would be built on the builders (i.e., native for x86_64* and i686, cross for aarch64* and armv[67]l*)

Yeah, right. But such builders do cross-compilation and can't run compiled binaries. That's why I made these internal tests.

Reply:
Ah, moreover. Builders do not include ''armv5te and armv5tel, armv6hf, armv7hf, mips,mipshf, mipsel, mipselhf, ppc and ppc64".

Reply:
> why? that's an important line with build options to not be lost inside urls and descriptions.
imho.

because that's what the style is. by putting it not in the first block xlint doesn't catch it but that's where it should go

> Builders do not include ''armv5te and armv5tel, armv6hf, armv7hf, mips,mipshf, mipsel, mipselhf, ppc and ppc64".

because these aren't currently supported archs. the profiles exist but we don't really support them

Reply:
> because that's what the style is.
Got it. Ok!

Reply:
Done.